cli/kafka/config.go

```diff
@@ -8,6 +8,14 @@ import (
 	"github.com/featurebasedb/featurebase/v3/dax"
 	"github.com/featurebasedb/featurebase/v3/idk"
 	"github.com/pkg/errors"
+	"github.com/spf13/viper"
+)
+
+// Kafka message encoding types supported - changes here should be
+// propogated to the Config struct and ValidateConfig error message
+const (
+	JSON string = "json"
+	Avro string = "avro"
```

Comment:
I would give these names specific to their intended use, so something like:
```
encodingTypeJSON = "json"
encodingTypeAvro = "avro"
```
Note a couple things:
- unexported because you don't use these outside of this package
- the "string" part is implied so you don't need it; a linter will complain.